theme.config.js

```diff
@@ -20,12 +22,22 @@ export default {
       <meta name="theme-color" content="#ffffff" />
       <meta name="viewport" content="width=device-width, initial-scale=1.0" />
       <meta httpEquiv="Content-Language" content="en" />
-      <meta name="description" content="Sound Docs" />
-      <meta name="og:description" content="Sound Docs" />
+      <meta name="apple-mobile-web-app-title" content="Sound Docs" />
+
+      {['og:site_name', 'og:title', 'twitter:title'].map((property) => (
+        <meta key={property} property={property} content={'Sound Docs'} />
+      ))}
+      {['description', 'og:description', 'twitter:description'].map((property) => (
+        <meta key={property} name={property} content={'Tools for empowering artists & collectors'} />
+      ))}
       <meta name="twitter:site:domain" content="www.sound.xyz" />
       <meta name="twitter:url" content="https://www.sound.xyz" />
-      <meta name="og:title" content="Sound Docs" />
-      <meta name="apple-mobile-web-app-title" content="Sound Docs" />
+
+      {/* 
+        TODO: Uncomment this after we've launched the new logo.
+      <meta name="twitter:image" content={SOCIAL_CARD_IMAGE} />
+      <meta property="og:image" content={SOCIAL_CARD_IMAGE} /> 
+      */}
```

Comment:
what image are we showing now? Maybe @ohjessica can help create with the old logo like we just did for main site `https://d2i9ybouka0ieh.cloudfront.net/website-content/twitter/new-logo-og.png`

Reply:
fixed: https://github.com/soundxyz/docs/pull/24/commits/7d5f83248eaecc6c62a4dc085043454e50237ae4